File: PlagueExternal.dll/GetPublicOrderEffect.cs
float __fastcall GetPublicOrderEffect(SPLocalDiseaseData localDisease)
{
  return (((localDisease.infectedPercent + localDisease.infectedPercent) / (localDisease.infectedPercent + 0.5))
               * (localDisease.localSeverity / 135.0))
       * (localDisease.localSeverity / 135.0);
}

File: PlagueLibrary/PlagueExternal/GetPublicOrderEffect.c
// Original function: GetPublicOrderEffect(__int64 a1)
float __fastcall GetPublicOrderEffect(__int64 ld)
{
  return (((ld.infectedPercent + ld.infectedPercent) / (ld.infectedPercent + 0.5))
               * (ld.localSeverity / 135.0))
       * (ld.localSeverity / 135.0);
}
